src/wiki/development/build-instructions.md

```diff
@@ -63,7 +64,6 @@ This is the preferred method of installation, and is suitable for packages.
 cmake -S . -B build \
    -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Since they're building as a Release either way, why not tell about `-DENABLE_LTO` as well? :)